test/composertest.py

```diff
@@ -93,8 +82,8 @@ def runCliTest(self, script):
         if self.sit:
             execute_params.insert(0, "COMPOSER_TEST_FAIL_FAST=1")
 
-        r = self.execute(execute_params)
-        self.assertEqual(r.returncode, 0)
+        returncode = self.execute(execute_params)
+        self.assertEqual(returncode, 0)
```

Comment:
`execute` returns a [`CompletedProcess`](https://docs.python.org/3/library/subprocess.html#subprocess.CompletedProcess) instance. It contains the return code in its `returncode` attribute.

Reply:
Nope it doesn't. As part of the original backport the code on the RHEL7 branch now uses `subprocess.call()` which returns an `int`. On master the code uses `subprocess.run()` which returns the `CompletedProcess` instance.

See the notes in the extended comment on `Don't override TestCase.run()` commit. 

I can revert to the same code on master but I will need to add pylint disable at the very least. It will also be a bigger change.

Reply:
That's super confusing. Why change it at all? `subprocess.run()` exists in RHEL 7's python3.

> - I have kept the other backport changes partly due to pylint and
  partly b/c they have been introduced with a single commit and
  require manually to check them out and revert them (these are mostly
  around how we call subprocess-es).

I'm sorry for the extra work, but this is not an excuse to keep it in the code.